.ci.yaml

```diff
@@ -3994,7 +3994,6 @@ targets:
         ["devicelab", "hostonly"]
       task_name: platform_channel_sample_test_windows
     scheduler: luci
-    bringup: true
```

Comment:
I validated (by temporarily making this a presubmit test in an earlier version of this PR) that this test passes with the changes here.